remappers/hostmetrics/network.go

```diff
@@ -0,0 +1,88 @@
+// Licensed to Elasticsearch B.V. under one or more contributor
+// license agreements. See the NOTICE file distributed with
+// this work for additional information regarding copyright
+// ownership. Elasticsearch B.V. licenses this file to you under
+// the Apache License, Version 2.0 (the "License"); you may
```

Comment:
Any reason to not use the migrated network.go file ?

Reply:
Same as above, I moved the file but git shows it as delete + add.